File: c_hash/lesson_1/task_2.cs
Console.WriteLine("Задача 4: Напишите программу, которая принимает на вход три числа и выдаёт максимальное из этих чисел.");
/* 

2, 3, 7 -> 7
44 5 78 -> 78
22 3 9 -> 22
*/

int a = 9;
int b = 10;
int c = 7;

if ((a > b) & (a > c)){
    Console.WriteLine($"max = {a}");
} else if ((a < b) & (b > c)){
    Console.WriteLine($"max = {b}");
} else {
    Console.WriteLine($"max = {c}");
};

File: prepair-to-gb/python_gb/hw4/task2.py
# Задача 2. Дан список случайных чисел. Создайте список, в который попадают числа, 
# описывающие случайную возрастающую последовательность. Порядок элементов менять нельзя.
# [1, 5, 2, 3, 4, 6, 1, 7] => [1, 2, 3] или [2, 7] или [4, 6, 7] и т.д.

from random import randint
my_list =  [randint(1, 10) for i in range(10)]
result = []

for i in range(len(my_list)):
        if my_list[i] == max(my_list[:i+1:]) and my_list[i] not in result:
            result.append(my_list[i])

print(f'{my_list} => {result}')